src/routes/general.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { Route } from 'react-router-dom'
 import App from '../App'
-import { Register } from '../components'
+import { HotelMoreDetails, Register } from '../components'
 import { Hotels } from '../components/views/mainPageView/Hotels'
 import { NoMatch } from '../components/NoMatch'
```

Comment:
Can you import NoMatch just from Components? Same with Hotels

Reply:
Done